src/analytics/private/metrics-common-workflow.ts

```diff
@@ -58,19 +74,18 @@ export function buildMetricsWidgetForWorkflows(scope: Construct, id: string, pro
     'service', MetricsService.WORKFLOW,
   ];
 
-  const loadEventsWorkflowAlarm = new Alarm(scope, id + 'loadEventsWorkflowAlarm', {
+  const loadEventsWorkflowAlarm = new Alarm(scope, id + 'loadEventWorkflowAlarm', {
```

Comment:
Should we rename it to `LoadDataWorkflowAlarm`?

Just see a single alarm here, do we share the workflow to load data to tables?

Reply:
no. each destination table has a workflow, there are 4 workflows.

Reply:
Will we have four alarms for four loading workflows?

Reply:
No. only 2 alarams, one for failed workflow, one for max file age.

Reply:
Will the failure of any table loading trigger the alarm?